.github/workflows/release.yml

```diff
@@ -65,7 +65,7 @@ jobs:
     runs-on: windows-latest
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9, 3.10]
```

Comment:
```suggestion
        python-version: [3.7, 3.8, 3.9, 3.10.0]
```

Reply:
why `.0`?

Reply:
seems that somehow gh actions detects 3.1 if i don't explicitly set the patch version

Reply:
the trick was to define as strings